planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -562,6 +561,16 @@ bool NormalLaneChange::getLaneChangePaths(
     *lane_change_parameters_);
 
   candidate_paths->reserve(longitudinal_acc_sampling_values.size() * lateral_acc_sampling_num);
+  const auto is_near_end_of_road = std::invoke([&]() {
```

Comment:
Use `is_near_end_of_current_lane` instead of `is_near_end_of_road` to specify the lanes

Reply:
changed in [c4c1405](https://github.com/autowarefoundation/autoware.universe/pull/4242/commits/c4c140511a6a695224eef58b828a45c232dcfab4)